cmd/server-main.go

```diff
@@ -134,13 +134,13 @@ func serverMain(ctx *cli.Context) {
 
 	// Get "json" flag from command line argument and
 	// enable json and quite modes if jason flag is turned on.
-	jsonFlag := ctx.Bool("json") || ctx.GlobalBool("json")
+	jsonFlag := ctx.IsSet("json") || ctx.GlobalIsSet("json")
 	if jsonFlag {
 		log.EnableJSON()
 	}
 
 	// Get quiet flag from command line argument.
-	quietFlag := ctx.Bool("quiet") || ctx.GlobalBool("quiet")
+	quietFlag := ctx.IsSet("quiet") || ctx.GlobalIsSet("quiet")
```

Comment:
I think you would need to move `json` and `quiet` flags handling to `handleCommonCmdArgs()`

Reply:
If we have to check if the quiet flag is set, so that we do not check for minio update, then it should be outside.

Reply:
Isn't update check common to gateway and server?

Reply:
Yes, we are doing it in both places currently